hdf5.spec

```
@@ -10,25 +10,22 @@
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: 1.10.5
-Release: 9.g07066a381e%{?dist}
+Version: 1.12.0
+Release: 1.g4aa4036562%{?dist}
```

If this is the official 1.12.0 release you don't want/need the git hash in the `Release:`:
```suggestion
Release: 1%{?dist}
```